JobConfig/reco/prolog.fcl

```diff
@@ -100,7 +100,8 @@ Reconstruction : {
 	HelixSeedCollections : [ "MHDeM" ]
         ComboHitCollection : "makeSH"
         StrawHitFlagCollection : "FlagBkgHitss:StrawHits"
```

Comment:
`FlagBkgHitss` is this a typo? 2 's'?

Reply:
good catch!  Not sure why, but this worked.  I'll investigate.

Reply:
This collection isn't currently being used so the config error was irrelevant.  I fixed it anyways.